harness/determined/deploy/local/cli.py

```diff
@@ -240,6 +253,12 @@ def deploy_local(args: argparse.Namespace) -> None:
                     default="determined",
                     help="name for the cluster resources",
                 ),
+                Arg(
+                    "--image-repo-prefix",
+                    type=str,
+                    default="determinedai",
+                    help="prefix for the master image",
+                ),
```

Comment:
What is the purpose of adding this one?

Reply:
1) cluster-up and agent-up both have these, and 2) the method this CLI calls `handle_master_up` actually takes in `args.image_repo_prefix` even though we never let it be specified.

Reply:
So, this cli is a public interface.

Do we want to offer an `--image-repo-prefix`?  Why not just offer e.g. a `--master-image` and an `--agent-image` or something more direct?  Or not add it to the public interface if we haven't thought out fully?

Reply:
yeah, i don't disagree that `master-image` and `agent-image` might be better. my reasoning for this was simply to make it more consistent with what the API already does. we already take in `image-repo-prefix` for the other CLI calls, and it seemed like a bug that we didn't have this here. before, we still technically accepted the argument, we would just default inside of the method, making the defaults not visible and not configurable by the user. i can remove it if you think it's not an overall gain.

Reply:
I'll trust your judgement on this.